staging/src/k8s.io/pod-security-admission/policy/check_sysctls.go

```diff
@@ -71,6 +72,7 @@ var (
 		"net.ipv4.tcp_syncookies",
 		"net.ipv4.ping_group_range",
 		"net.ipv4.ip_unprivileged_port_start",
+		"net.ipv4.ip_local_reserved_ports",
```

Comment:
this modifies the existing instance of the sysctl check, which we don't want to do... changing a pod-security-policy should only happen in a versioned way, and should start with a proposed update to https://kubernetes.io/docs/concepts/security/pod-security-standards/

after sig-node / sig-network ack this is a safe sysctl to add, we can figure out how to phase this in here

Reply:
/hold

> after sig-node / sig-network ack this is a safe sysctl to add, we can figure out how to phase this in here

Thanks for the information.